jormungandr-integration-tests/src/mock/mod.rs

```diff
@@ -5,17 +5,17 @@ extern crate grpc;
 extern crate hex;
 extern crate protobuf;
 
+#[macro_use]
 pub mod client;
 pub mod convert;
 pub mod proto;
 pub mod server;
 
+use chain_core::mempack::{ReadBuf, Readable};
```

Comment:
Commingling `use` of external things and `pub use` of reexports is confusing. This line is better where it was before.